common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -596,7 +605,8 @@ class SplitModuleCourseTests(SplitModuleTest):
     Course CRUD operation tests
     '''
 
-    def test_get_courses(self):
+    @patch('xmodule.tabs.CourseTab.from_json', side_effect=mock_tab_from_json)
```

Comment:
Did you really mean `side_effect` here? If you just want to replace `from_json` with `mock_tab_from_json`, then you don't need to name the argument.